src/app/shared/filter-bar/filter-bar.component.css

```diff
@@ -1,22 +1,67 @@
-.dropdown-filters .mat-form-field {
+.filters .mat-form-field {
   margin: 8px;
   font-size: 14px;
   max-width: 20%;
   width: 14%; /* depends on number of filters*/
 }
 
-.search-bar {
-  width: 80%;
+ul {
+  margin: 8px;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  list-style-type: none;
 }
 
-.dropdown-filters {
+.search-bar {
   width: 100%;
 }
 
+.filters {
+  width: 57%;
+  display: flex;
+}
+
 .label-filter-grid-tile {
   width: 100%;
 }
 
-.submit-button {
-  margin-left: 8px;
+.menu {
+  padding: 0 10px;
+  border-left: 1px solid #eee;
+  display: none;
+  cursor: pointer;
+
+  @media (max-width: 1148px) {
```

Comment:
At about 1200-1300 px range there is overlapping of components
![image](https://github.com/user-attachments/assets/ae490e7c-6635-4a79-a945-13025c8184db)
Would it be better to use a more standard, larger breakpoint of 1400px?
Refer to https://getbootstrap.com/docs/5.0/layout/breakpoints/ for some standard breakpoints.